platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -99,10 +99,20 @@
 	 */
 	public static native void hover(int type, int x, int y);
 
+	/**
+	 * Forward mouse pressed events from the main thread to the GL thread.
+	 */
+	public static native void mouse_pressed(int button, int x, int y, boolean pressed);
+
+	/**
+	 * Forward  mouse moved while pressed events from the main thread to the GL thread.
+	 */
+	public static native void mouse_moved_pressed(int button, int x, int y);
```

Comment:
@thebestnom So I'm little bit confused as to the use-case for this method. 
In the other PR, you mentioned it was for dragging events, but can't that information be inferred using the `mouse_pressed` method? For example, if a button is pressed at `(x1, y1)` and released at `(x2, y2)`, the logic can infer that there was a drag event between the two locations.

Reply:
no, mouse pressed only fire on mouse down and mouse up.
drag event is a little bit different by nature by using a different godot event

Reply:
`InputEventMouseMotion` is for drag and hover while `InputEventMouseButton` is for press and release but what I managed to understand

Reply:
but it can be merged with hover but than the function name will make no sense so I decided not to

Reply:
Also because the Android event is completly different it feels better for me, think about it again and give me final decision, if you still think to merge the events I will do it